packages/venia-concept/src/util/makeUrl.js

```diff
@@ -1,8 +1,8 @@
-// ensure there's exactly one slash between segments
-const joinUrls = (base, url) =>
-    (base.endsWith('/') ? base.slice(0, -1) : base) +
-    '/' +
-    (url.startsWith('/') ? url.slice(1) : url);
+const absoluteUrlRE = /^(?:https?|data|file):/;
+
+// If the root template supplies the backend URL at runtime, use it directly
```

Comment:
Why pull this value at runtime instead of baking it into the build w/ `process.env.MAGENTO_BACKEND_URL`?

Reply:
I try to avoid embedding highly volatile strings into the bundles, so that we can reuse bundles between deployments. Any env var that can vary from individual deployment to deployment should maybe come out of runtime.